x11-wm/oroborus/oroborus-2.0.20-r1.ebuild

```diff
@@ -22,3 +21,5 @@ DEPEND="${RDEPEND}
 	x11-base/xorg-proto"
 
 DOCS=( AUTHORS ChangeLog example.${PN}rc README TODO )
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
```

Comment:
put this before `DOCS` (`src_prepare` vs `src_install`)

Reply:
Done